packages/core/src/compiler/Style.ts

```diff
@@ -1286,23 +1062,47 @@ const combine = (s1: Subst, s2: Subst): Subst => {
 // TODO check for duplicate keys (and vals)
 // (x) operator combines two lists of substitutions: [subst] -> [subst] -> [subst]
 // the way merge is used, I think each subst in the second argument only contains one mapping
```

Comment:
Does this `TODO` still apply?

Reply:
Duplication of keys should be prohibited by Subst inherently because it is a JS/TS object. Was there some other implementation of Subst that allowed this?

Duplication of values is checked by another function

Reply:
Not sure ¯\_(ツ)_/¯ but in any case, if this no longer applies then it's probably best to remove the comment?

Reply:
Sounds good I'll fix that.